phone/src/apps/twitter/components/AddTweetModal.tsx

```diff
@@ -90,9 +91,9 @@ export const AddTweetModal = () => {
   const handleMessageChange = useCallback((message) => setMessage(message), [setMessage]);
 
   useEffect(() => {
-    window.addEventListener('keydown', _handleEscape, true);
-    return () => window.removeEventListener('keydown', _handleEscape);
-  });
+    window.addEventListener('keyup', _handleEscape, true);
```

Comment:
Does this close the whole phone as well?

Reply:
No